samples/benchmarks/coremark/README.rst

```diff
@@ -0,0 +1,170 @@
+.. _coremark_sample:
+
+CoreMark
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to run the `CoreMark®`_ benchmark to evaluate performance of a core.
+
+.. _CoreMark®: https://www.eembc.org/coremark/
+
+Requirements
+************
+
+The sample supports the following development kits:
+
++--------------------------------+-----------+------------------------------------------------+------------------------------------+
+|Hardware platforms              |PCA        |Board name                                      |Build target                        |
```

Comment:
Again, given the nature of the CoreMark (it only utilizes CPU core) I would strongly suggest removing all the references to particular boards. At best I would accept a comment "Tested on XXX", but given it is a platform-agnostic benchmark we should have it functionally tested in QEMU on all supported architectures. And then interested users will be able to run it on their platform of choice.

Reply:
will remove, agree